src/views/preview/comment/top-level-comment.jsx

```diff
@@ -33,6 +34,12 @@ class TopLevelComment extends React.Component {
         this.props.onDelete(commentId, this.props.id);
     }
 
+    handleReportReply (commentId) {
+        // Only apply topLevelCommentId for reporting replies
```

Comment:
sorry, i don't follow this comment. what does "apply" mean in this context?

Reply:
I can change the comment. It means it is adding topLevelCommentId to the call to `onReport` as an additional param. I guess I meant it in a `Function.apply` sense.

Reply:
This could be clearer if `commentId` was renamed to `replyId` here.